.env.test

```diff
@@ -3,6 +3,10 @@ FLASK_APP=todo_app/app
 FLASK_ENV=development
 SECRET_KEY=12345
 
+# Logging
+LOG_LEVEL=DEBUG
+LOGGLY_TOKEN=123
```

Comment:
We don't really want tests to try logging anything. Unless you want to mock the logger and assert that your code is trying to log things correctly.

So don't set LOGGLY_TOKEN, and your app.py code happily skips over the loggly configuration.